lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -158,16 +158,28 @@ def higher_item
           higher_items(1).first
         end
 
+        def higher_items_relation
+          ids = higher_items.map(&:id)
+          acts_as_list_list.where(id: ids).reorder("#{quoted_position_column_with_table_name} DESC")
+        end
+
+        def lower_items_relation
+          ids = lower_items.map(&:id)
+          acts_as_list_list.where(id: ids).reorder("#{quoted_position_column_with_table_name} ASC")
+        end
+
         # Return the next n higher items in the list
         # selects all higher items by default
         def higher_items(limit=nil)
-          limit ||= acts_as_list_list.count
+          limit ||= acts_as_list_list.length
```

Comment:
Yep, that was my intent, since I was trying to reduce database calls. I wasn't considering cases where people would use very large lists.